src/loader-components/ExampleTemplate.module.scss

```diff
@@ -49,6 +50,21 @@
     display: flex;
     outline: none;
     padding: 0.625rem;
+
+    &.is-selected {
+      --terra-dev-site-example-template-chevron-color: --terra-dev-site-example-template-selected-code-toggle-color;
```

Comment:
What's this?

Reply:
Originally it was so that the chevrons matched the text color when the button was selected or highlighted, but I think the better option would be to just remove the [color specification for the chevrons](https://github.com/cerner/terra-dev-site/blob/c35fa4caeace3ff1ad2d72fa7248e406cd632be1/src/loader-components/ExampleTemplate.module.scss#L55) all together.

Reply:
Removed chevron coloring: [f086e80](https://github.com/cerner/terra-dev-site/pull/281/commits/f086e8097bc5393e9187cd5ca9efb848e1dcb773).